magento/adobe-stock-integration
# Internet explorer improvements && styles improvements

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1219: Image extension is cut in Media Gallery [Internet Explorer]
2. magento/adobe-stock-integration#1162: Long image names are not cut in Media Gallery on Internet Explorer and Edge web browsers
3. magento/adobe-stock-integration#1167: Wrong button order when an image is selected [Internet Explorer]

### Manual testing scenarios (*)
1. Open Magento instance on _Microsoft Internet Explorer_ OR _Edge_ web browser
2. Go to **Content** - **Media Gallery**
3 Verify the previously uploaded images name in the Media Gallery 

**Search Adobe Stock** is the last button, **Add Selected** is the last but one 
Observe the images names and extensions
